src/components/References.js

```diff
@@ -0,0 +1,23 @@
+import React, { Fragment } from 'react';
+import generateKey from '@tammiec/generatekey';
+
+import './References.scss';
+
+const References = ({ data, ...props }) => {
+
+  const references = data.map(ref => (
```

Comment:
Be aware with short named parameters like `ref`, prefer complete descriptive name parameters like `reference`